components/builder-originsrv/src/data_store.rs

```diff
@@ -1813,6 +1825,83 @@ impl DataStore {
         oin.set_names(names);
         oin
     }
+    pub fn create_origin_secret(
+        &self,
+        os: &originsrv::OriginSecretCreate,
```

Comment:
Shouldn't this be `osc`?  :trollface: KIDDING KIDDING

Reply:
:shaved_ice: